website/data/cdktf-nav-data.json

```
@@ -133,6 +137,10 @@
       {
         "title": "Go",
         "path": "api-reference/go"
+      },
+      {
+        "title": "Provider API References",
```

To match the other pages :-) 

```suggestion
        "title": "Providers",
```